cnxarchive/database.py

```diff
@@ -112,6 +113,60 @@ def initdb(settings):
             for schema_part in get_schema():
                 cursor.execute(schema_part)
 
+            # If you are connecting to a database that is not localhost,
+            # don't initalize with virtualenv
+            db_dict = dict(p.split('=') for p in db_connection.dsn.split())
+            if 'host' not in db_dict.keys() or ('host' in db_dict.keys() and
```

Comment:
I think this can be simplified to `if db_dict.get('host', 'localhost') == 'localhost':`